.github/workflows/dispatch_image_builder.yaml

```diff
@@ -0,0 +1,92 @@
+on:
+  repository_dispatch:
+    types:
+      - release
```

Comment:
Will you remove https://github.com/outscale/cluster-api-provider-outscale/blob/main/.github/workflows/image-builder.yaml ?

Reply:
Yes

Reply:
Yes

Reply:
I will remove it in an other PR (if you ok ?) when there will be a new release of kubernetes 1.26.2.

Reply:
Ok 👍